Makefile.shared

```diff
@@ -31,29 +30,27 @@ endif
 ifneq (,$(SDKROOT))
 	ifneq (,$(OVERRIDE_SDKROOT))
 		ifneq (default,$(OVERRIDE_SDKROOT))
-			XCODE_OPTIONS := $(XCODE_OPTIONS) SDKROOT=$(OVERRIDE_SDKROOT)
+			BUILD_WEBKIT_OPTIONS := $(BUILD_WEBKIT_OPTIONS) --sdk=$(OVERRIDE_SDKROOT)
```

Comment:
This is a bit surprising, from a cursory inspection of code `BUILD_WEBKIT_OPTIONS` seems like it's meant to be an input parameter (options passed to `build-webkit`).

Reply:
Great catch. `BUILD_WEBKIT_OPTIONS` should probably be named something like `BUILD_WEBKIT_ARGS` to indicate this, but there's no point in breaking existing behavior now. I'll move all the flags passed by the Makefile into their own variable.